docs/application_packages.md

```diff
@@ -451,7 +451,7 @@ Application Package needs the classpath in a specific order.
 ### Logging configuration
 
 Just like other Java projects, you can change the logging configuration
-by having your log4j.properties under src/main/resources.  For example,
+by updating your log4j.props under src/main/resources.  For example,
 if you have the following in src/main/resources/log4j.properties:
```

Comment:
This one says log4j.properties although the file was log4j.props. In any case I prefer properties to props as stated before.